src/genegraph/source/graphql/schema/proband_evidence.clj

```diff
@@ -47,4 +47,7 @@
                                 :path [:sepio/detection-method]}
             :ethnicity {:type :Resource
                         :description "The ethnicity of the proband."
-                        :path [:sepio/ethnicity]}}})
+                        :path [:sepio/ethnicity]}
+            :paternity_maternity_confirmed {:type 'String
+                                            :description "Paternity/maternity confirmed."
+                                            :path [:sepio/paternity-maternity-confirmed]}}})
```

Comment:
What are the possible values of this?

Reply:
"Yes", "No", "none" when included.

Reply:
This feels like some technical debt; but we're under time pressure and it solves the immediate problem. Doing it this way leaves open the possibility of doing it better in the future, so I'm OK with this.